integrations/groq/src/index.ts

```diff
@@ -136,20 +150,6 @@ const languageModels: Record<ModelId, llm.ModelDetails> = {
       maxTokens: 32768,
     },
   },
-  'gemma-7b-it': {
```

Comment:
note: Groq has "deprecated" (they used it as an euphemism, as they are actually discontinuing it with very short notice) the Gemma 7B (first version) model and they'll be removing it on Dec 18th, so I'm removing it from the model list completely.

This means that bots using the older integration versions will start to fail if they're using this model. Based on our LLM stats [published here](https://mediafiles.botpress.cloud/public/llm-stats.json), it seems only 12 requests have been made to this model in the current month so far, so I think it's not a huge issue, as users can just use the better/newer Gemma 2 7B model instead (it's token prices are about 2x more expensive, though), or worst case we do a forceful remapping of `generateContent actions calls using the `groq:gemma-7b-it` model to use `groq:gemma2-9b-it` instead.

<img width="815" alt="image" src="https://github.com/user-attachments/assets/d5a59001-456f-44b4-819c-dc417e52ae0c" />

FYI @slvnperron